scripts/scil_compute_todi.py

```diff
@@ -33,39 +35,39 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
                                 formatter_class=argparse.RawTextHelpFormatter)
 
-    p.add_argument('in_bundle',
+    p.add_argument('input_tracto',
                    help='Input streamlines file.')
 
     add_reference_arg(p)
 
     p.add_argument('--sphere', default='repulsion724',
-                   help='sphere used for the angular discretization.')
+                   help='sphere used for the angular discretization. '
+                        '[%(default)s]')
 
     p.add_argument('--mask',
                    help='Use the given mask.')
 
     p.add_argument('--out_mask',
                    help='Mask showing where TDI > 0.')
 
-    p.add_argument('--out_lw_tdi',
-                   help='Output length-weighted TDI map.')
+    p.add_argument('--out_tdi',
+                   help='Output Track Density Image (TDI).')
 
-    p.add_argument('--out_lw_todi',
-                   help='Output length-weighted TODI map.')
+    p.add_argument('--out_todi_sf',
+                   help='Output TODI, with SF (each directions\n'
+                        'on the sphere), require a lot of memory)')
 
-    p.add_argument('--out_lw_todi_sh',
-                   help='Output length-weighted TODI map, with SH '
-                        'coefficient.')
+    p.add_argument('--out_todi_sh',
+                   help='Output TODI, with SH coefficients.')
 
-    p.add_argument('--sh_order',
-                   type=int, default=8,
-                   help='Order of the original SH.')
+    p.add_argument('--sh_order', type=int, default=8,
+                   help='Order of the original SH. [%(default)s]')
 
-    p.add_argument('--sh_normed', action='store_true',
-                   help='Normalize SH.')
+    p.add_argument('--normalize_per_voxel', action='store_true',
+                   help='Normalize each SF or SH at each voxel.')
 
-    p.add_argument('--smooth', action='store_true',
-                   help='Smooth todi (angular and spatial).')
+    p.add_argument('--smooth_todi', action='store_true',
+                   help='Smooth TODI (angular and spatial).')
```

Comment:
please add the defaults for the boolean options too. [false]

Reply:
@arnaudbore is it common to add default to "action='store_true',"

Reply:
I want the user to know it is a boolean option. Right now impossible to know with --smooth_todi needs to be called with a float or a true or false. Just find a way for the user to call it with 
--smooth_todi true 
and not --smooth_todi 0.25 or whatever

Reply:
done